driver/ppm_events_public.h

```diff
@@ -433,6 +433,14 @@ along with sysdig.  If not, see <http://www.gnu.org/licenses/>.
 #define PPM_SETVAL		(1 << 12)
 
 /*
+ * Access flags
+ */
+#define PPM_F_OK            (0)
```

Comment:
Usually we let these flags start from 1, also probably your patch to sinsp is not needed if they start from 1.